torch/csrc/inductor/aoti_runner/model_container_runner_cuda.h

```diff
@@ -9,7 +9,8 @@ namespace torch::inductor {
 // NOTICE: Following APIs are subject to change due to active development
 // We provide NO BC guarantee for these APIs
 // NOLINTNEXTLINE(cppcoreguidelines-special-member-functions)
-class TORCH_API AOTIModelContainerRunnerCuda : public AOTIModelContainerRunner {
+class TORCH_CUDA_CPP_API AOTIModelContainerRunnerCuda
```

Comment:
@desertfire  Her I changed `TORCH_API` to `TORCH_CUDA_CPP_API` for the following reason:
With `TORCH_API` the ciflow/trunk job cuda windows build failed with error message when linking libtorch_cuda: 
```
2024-12-06T06:09:28.8270765Z model_container_runner_cuda.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) const torch::inductor::AOTIModelContainerRunnerCuda::`vftable'" (__imp_??_7AOTIModelContainerRunnerCuda@inductor@torch@@6B@) referenced in function "public: __cdecl torch::inductor::AOTIModelContainerRunnerCuda::AOTIModelContainerRunnerCuda(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,unsigned __int64,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (??0AOTIModelContainerRunnerCuda@inductor@torch@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@_K00@Z)
```
which shows the `vftable` is an `__declspec(dllimport)` symbol.
Then I found that `TOCH_API` is expressed as `C10_IMPORT` for lib_cuda, we should use `TORCH_CUDA_CPP_API` here.